apiary.apib

```diff
@@ -564,6 +564,75 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             errors: "Some error message"
         }
 
+## Mentors | Collection [/api/v1/mentors]
+
+### List All Mentors Available [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "id": 42,
+                "email": "aunta@howe.net",
+                "zip": "97201",
+                "first_name": "Boris",
+                "last_name": "Hackett",
+                "slack_name": "Han Solo",
+                "mentor": true,
+                "verified": false,
+                "role": nil
+            },
+            {
+                "id": 43,
+                "email": "claude_predovic@block.net",
+                "zip": "97201",
+                "first_name": "David",
+                "last_name": "Powlowski",
+                "slack_name": "Poe Dameron",
+                "mentor": true,
+                "verified": false,
+                "role": nil
+            }
+        ]
+
+## Mentor | Show [/api/v1/mentors/{id}]
+
++ Parameters
+
+    + id (integer, required) - Identification of mentor request in the form of an integer
+
+### Show a single mentor [GET]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
+    + Body
```

Comment:
Since the `id` is part of the passed url, this endpoint wouldn't actually have a body section.